src/main/resources/application.yaml

```diff
@@ -205,3 +204,5 @@ flyway:
 
 process-payments:
   enabled: ${PROCESS_PAYMENTS_ENABLED:true}
+
+use-wrapping-zip: ${USE_WRAPPING_ZIP:true}
```

Comment:
this env var should be ideally defined in values yaml and default value should be removed

Reply:
perhaps I'll just remove the env var then, we currently don't need to change it